src/openzaak/components/catalogi/tests/admin/test_eigenschap_admin.py

```diff
@@ -49,3 +51,42 @@ def test_validation_length_comma_separated(self):
 
         new_specificatie = EigenschapSpecificatie.objects.get()
         self.assertEqual(new_specificatie.lengte, "5,3")
+
+
+@tag("issue-1042")
```

Comment:
```suggestion
@tag("gh-1042")
```
let's see if we can stick to the pattern of tags used in other places :P